firmware/src/sensors/sensors_task.cpp

```diff
@@ -172,7 +172,7 @@ void SensorsTask::run()
 
                     strain_reading_raw = strain.get_units(1);
 
-                    if (abs(last_strain_reading_raw_ - strain_reading_raw) > 1000)
+                    if (abs(last_strain_reading_raw_ - strain_reading_raw) > 2000)
```

Comment:
any specific reason why we changed this value  ?

Reply:
Was getting some unwanted discards, shouldve been changed on different PR.

Reply:
Ok for now, but let's a todo on the code that we want to replace that number with a 'learned' one based on the physical attributes of the knob